google-cloud-core-grpc/src/main/java/com/google/cloud/grpc/GrpcTransportOptions.java

```diff
@@ -72,6 +75,26 @@ public void close(ScheduledExecutorService instance) {
       };
 
   /**
+   * Returns a string value for x-goog-api-client HTTP header. The header is used to report version
+   * of the client and its protocol-specific dependencies.
+   *
+   * For internal use.
+   *
+   * @param libraryVersion version of the google-cloud-java library
+   * @return value of x-goog-api-client HTTP header, which should be provided with each request
+   */
+  @InternalApi
+  public String getXGoogApiClientHeader(String libraryVersion) {
+    return String.format(Locale.US,
+        "gl-java/%s %s/%s gax/%s grpc/%s",
```

Comment:
I would say yes. Gapic is basically codegen clients, which depend on grpc. 
Currently for java client the "gapic/" part of header does not make sense unless at least some part of the client was generated by toolkit. Spanner currently does not have generated parts in it (there is a plan to migrate to gapic), that is why the gapic part is not there in the header.

More details about gax.

In java we recently split our gax into just "gax" and "gax-grpc". All clients (manual, semi-manual and generated) use gax (because it has some fundamental classes, like RetrySettings for example). Not all clients use gax-grpc (for example datastore does not use it).
I discussed that with @omaray, and as result we decided to put gax header if at least gax (not gax-grpc) is used. Note, currently gax and gax-grpc are always released together, so gax version always have an equal corresponding gax-grpc (so for clients, which use both, knowing version of gax is enough).